crates/bevy_ui/src/render/mod.rs

```diff
@@ -267,24 +357,21 @@ pub(crate) fn resolve_border_thickness(value: Val, parent_width: f32, viewport_s
     }
 }
 
-pub fn extract_uinode_borders(
+pub fn extract_uinode_border(
     mut commands: Commands,
     mut extracted_uinodes: ResMut<ExtractedUiNodes>,
     windows: Extract<Query<&Window, With<PrimaryWindow>>>,
     ui_scale: Extract<Res<UiScale>>,
     uinode_query: Extract<
-        Query<
-            (
-                &Node,
-                &GlobalTransform,
-                &Style,
-                &BorderColor,
-                Option<&Parent>,
-                &ViewVisibility,
-                Option<&CalculatedClip>,
-            ),
-            Without<ContentSize>,
```

Comment:
The reason this system has the `Without<ContentSize>` filter is because of two problems:

1. The size of a node with `ContentSize` is determined by it's `ContentSize` and may not include space for the borders IIRC. This could result in a messed up looking result with the borders overflowing the node or something.

2. Text is positioned in the node relative to the top left corner. So if you have a bordered text node the text will drawn over the border not inside it, like you might expect. It's complicated to fix this atm because (as well as issue 1) the text layout systems would need to aware of the border geometry. 

Outlines aren't a problem though as they are drawn outside of the UI node's bounds.

Reply:
Okay, I don't fully understand but I'm not gonna test it myself so I'll take your word. It's not necessary for this PR anyways